easycv/selectors.py

```diff
@@ -0,0 +1,134 @@
+from matplotlib.widgets import RectangleSelector, EllipseSelector
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+from easycv.io.output import prepare_image_to_output
+
+mpl.use("Qt5Agg")
+
+
+def toggle_selector(event):  # Bindings to close the matplotlib window
+    if event.key in ["Q", "q"] and toggle_selector.S.active:
+        toggle_selector.S.set_active(False)
+    if event.key in ["A", "a"] and not toggle_selector.S.active:
+        toggle_selector.S.set_active(True)
```

Comment:
Bindings should be explained in the module intro since they will be common to all selectors